java-squid/src/main/java/org/sonar/java/ast/parser/TreeFactory.java

```diff
@@ -466,44 +398,30 @@ public EnumConstantTreeImpl newEnumConstant(
     Optional<ClassTreeImpl> classBody,
     Optional<InternalSyntaxToken> semicolonToken) {
 
+    // FIXME SONARJAVA-547 Handle annotations
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1134)